README.md

```diff
@@ -18,7 +18,7 @@ The base URL for all API operations is  https://api.upcloud.com/ and require bas
 
 ## Requirements
 
-Using this library requires the PHP version 5.5 and later.
+Using this library requires the PHP version 7.2 and later.
```

Comment:
If travis is set to test 5.6, 7.0, 7.1 then this library should be able to run in those as well. Not just 7.2 ->